django/conf/global_settings.py

```diff
@@ -543,6 +543,8 @@ def gettext_noop(s):
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
+CSRF_SESSION_KEY = 'csrftoken'
```

Comment:
The only argument for configurability I see is backwards-compatibility if someone uses a session variable of the name we choose.